### PR TITLE
feat(website): alpha-2 testnet docs

### PR DESCRIPTION
**Pull request by dionysuzx**
please do not click "merge when ready", but take a look at the PR and feel free to approve. thanks

link: https://website-git-askja-release-taikoxyz.vercel.app/

### CODE REVIEW

File: packages/website/pages/docs/guides/run-a-node.mdx [hunk 1]
```diff
@@ -40,14 +40,22 @@ Then, open the `.env` file in your preferred text editor:
...
 </Tabs>
 
-Finally, set the following environment variables to configure your node: 
+Finally, set the following environment variables:
+  - L1_ENDPOINT_HTTP
+  - L1_ENDPOINT_WS
+
+> **Note:** You can get a Sepolia L1 endpoint from a few places, [Alchemy](https://www.alchemy.com/) is one popular example.
+
+### Enable your node as a prover (optional)
```
Comment (davidtaikocha) [inline packages/website/pages/docs/guides/run-a-node.mdx:49]: Shall we also put a reminder here that only the first prover can get the reward, and others will be rejected by the protocol smart contract?

**Review by davidtaikocha — DISMISSED** [review]

File: packages/website/pages/docs/guides/run-a-node.mdx [hunk 2]
```diff
@@ -40,14 +40,22 @@ Then, open the `.env` file in your preferred text editor:
   <Tab>```sh notepad .env ```</Tab>
 </Tabs>
 
-Finally, set the following environment variables to configure your node: 
+Finally, set the following environment variables:
```
Comment (davidtaikocha) [inline packages/website/pages/docs/guides/run-a-node.mdx:43]: Also remind people that if they have run alpha-1 node before, they need `docker compose down -v`?

Comment (d1onys1us) [reply packages/website/pages/docs/guides/run-a-node.mdx:45]: addressed.

File: packages/website/pages/docs/guides/run-a-node.mdx (hunk 1 above)
Comment (d1onys1us) [reply packages/website/pages/docs/guides/run-a-node.mdx:52]: done

**Review by davidtaikocha — APPROVED** [review]

**Review by mfinestone — APPROVED** [review]

**Pull request merged**